README.md

```diff
@@ -149,6 +149,8 @@ sudo apt-get install    \
 **Note:** LLDB currently requires at least `swig-1.3.40` but will successfully build
 with version 2 shipped with Ubuntu.
 
+**Note:** For Ubuntu 20.04, use `libpython2-dev` in place of the libpython-dev package above.
```

Comment:
This is because 20.04 removed their EOL'd python 2 packages, right? Can we start pointing people towards installing python 3 instead?

Reply:
It is true that python2 has been depreciated, but we cannot point people to python3 until all of the other libraries that are pulling in python2 packages have been migrated (a non-trivial move).

Reply:
Are you saying that `apt` won't automatically fetch python2 on demand then? Or is there additional work *we* need to do to support python 3?

Reply:
the existing command will fail because 20.04 has removed libpython-dev from the package list.  replacing the package name with libpython2-dev will pull down the old libraries, at which point swift will continue working.

in order to move to fully python3, the various interdependent libraries that pull in python2 bits will all have be moved to python3 equivalents and any existing code that relies on said libraries will need to be moved to python3 as well.  this will probably be some work.

Reply:
Should we just point people towards installing `libpython2-dev` in all cases, then?

Reply:
no, because 18.04 and prior use libpython-dev instead --> 

```
sudo apt-get install libpython2-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libpython2-dev
```

Reply:
Alright then, I'll take this.